back-office/wishlist.md

```diff
@@ -0,0 +1,350 @@
+
+Functional rules: 
+A user does not need to be logged in to view the shared link > an "ephemeral" page is created with a product listing outside the users account, without the option to delete or modify the list > model need 
+
+## Feature 3: Bookmarks - Back office
+### US 8: Set up the module and view statistics in back office 
+##### As a seller, I can modify the wording in back office 
+- Name of my Favorites section 
+- Title 
+- Create a list CTA
+
+A multi-language field for each field allows to translate them
+
+![wishlist screen 21](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-21.png)
+
+##### As a seller, I can view statistics regarding favorites in back office 
+
+![wishlist screen 22](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-22.png)
+
+available statistics: 
+Provide a cache for queries with calculations for large catalogs! (some sellers have more than 30,000 products)
+
+Here are the caches you need, depending on the timing: 
+-	All time & Year: 1 month cache
+-	Month: one week cache 
+-	Day: one day cache 
+
+A "Refresh" CTA; when a user changes the timing, they will have to click on Refresh to update their data. 
+
+1.	A top 10 of the most bookmarked products over a given period (via the PREFIX_wishlist_product_basket table)
+
+For each product, you should provide: 
+- Product image
+- a clickable product title that refers the user to the product sheet in back office
+- product reference 
+- Main category
+- Price excl. VAT 
+- Quantity available 
+- a conversion rate (= the number of times the product has been purchased after being listed as a favorite) 
+
+You can display the results: 
+-	for that day, 
+-	for the current month 
+
+PS: as opposed to the model, no filter is needed for the year and "All time" because the loading time is too long
+
+2.	Total number of created and active lists (still online) 
+
++	Retrieve customers' favorites lists to put them in the customer page > update the layout 
+
+## Multistore behavior 
+If the multistore feature is enabled and in the case of multi-shops in the same "default" group where the customers, the available quantity for sale and the orders are not shared: 
+- Wishlists will be different between stores in this group and will not be shared
+- the statistics table in the back office will be different depending on if  you are in shop A or shop B. It will not be a question of having global statistics including data from shop A and shop B.
+
+If the multistore feature is enabled and in the case of multi-shops in the same group where the customers, the available quantity for sale and the orders are shared: 
+- even if there is a sharing of customers accounts and products quantity between the shops, the visitor will not share his wishlists: he will have specific wishlists according to the shop he is in. 
+- the statistics are global statistics: include data from shops belonging to the same group 
```

Comment:
I think if we are in context "Group of shop" yes we need global statstics but if I'm in the context shop A it's better to see the stats of the shop A and not all the group even if I share product and customer.